.golangci.yml

```diff
@@ -121,6 +120,7 @@ linters-settings:
   gosec:
     excludes:
       - G107 # Url provided to HTTP request as taint input https://securego.io/docs/rules/g107
+      - G115 # TODO Enable this ruleset in a follow-up PR
```

Comment:
Disable majority of newly introduced linting errors with the golangci / golang version bump